src/SFA.DAS.LevyTransferMatching.Web/Views/Applications/ApplicationStatus.cshtml

```diff
@@ -0,0 +1,48 @@
+﻿@using SFA.DAS.LevyTransferMatching.Domain.Extensions
+
+@model SFA.DAS.LevyTransferMatching.Web.Models.Applications.ApplicationStatusViewModel
+
+@{
+    ViewData["Title"] = $"Your {(Model.IsNamePublic ? Model.EmployerAccountName : "opportunity")} ({Model.EncodedOpportunityId}) application details";
+}
+
+<a href="@Url.Action("GetApplications")" class="govuk-back-link">Back</a>
+<main class="govuk-main-wrapper govuk-main-wrapper--auto-spacing" id="main-content">
+
+    <div class="govuk-width-container form-entry">
```

Comment:
Can we take out this `<div>` there is already a `govuk-width-container` in the layout file. When we have more than 1 of these, we get extra padding/margin on smaller sized screens

Reply:
Thanks @jamesmking - removed this extra div 👍